yt_dlp/extractor/neteasemusic.py

```diff
@@ -253,12 +253,15 @@ def _real_extract(self, url):
             'formats': formats,
             'alt_title': '/'.join(traverse_obj(info, (('transNames', 'alias'), ...))) or None,
             'creator': ' / '.join(traverse_obj(info, ('artists', ..., 'name'))) or None,
+            'album_artist': ' / '.join(traverse_obj(info, ('album', 'artists', ..., 'name'))) or None,
             **lyric_data,
             **traverse_obj(info, {
                 'title': ('name', {str}),
                 'timestamp': ('album', 'publishTime', {self.kilo_or_none}),
-                'thumbnail': ('album', 'picUrl', {url_or_none}),
                 'duration': ('duration', {self.kilo_or_none}),
+                'album': ('album', 'name', {str_or_none}),
+                'thumbnail': ('album', 'picUrl', {str_or_none}),
```

Comment:
Is there a reason you changed this to `str_or_none`?

Reply:
> Is there a reason you changed this to `str_or_none`?
This was my mistake, thank you for reminding me of this.

Reply:
When changing it back, feel free to move it back up as to not have an unneccesary diff